src/Amalgam/string/StringManipulation.h

```diff
@@ -13,9 +13,16 @@ namespace StringManipulation
 	std::string NumberToString(size_t value);
 
 	//removes the first word from str and return the removed word
+	//if strip_word true, remove char_to_strip from beginning and end of returned word
 	std::string RemoveFirstWord(std::string &str);
 
-	std::vector<std::string> Split(std::string &s, char delim);
+	//splits a string by given delimiter
+	std::vector<std::string> Split(std::string &s, char delim = ' ');
+
+	//separates the argument string in a cross-platform manner and returns an appropriate vector of strings
```

Comment:
I would remove "cross-platform manner" here. I don't think it adds any information, anything outside of "PlatformSpecific" should be cross platform by definition.